src/components/loaders/DownloaderItem/Downloader.scss

```diff
@@ -0,0 +1,58 @@
+@import '../../../styles/_partials/index';
+
+.DownloaderOverlay {
+	z-index: 10;
```

Comment:
This change to z-index fixes a bug where some modals were appearing behind the overlay. E.g. when adding a site then trying to delete it.